test/e2e/main_test.go

```diff
@@ -64,14 +64,6 @@ func TestMain(m *testing.M) {
 					eventingtestlib.FeatureRedelivery,
 					eventingtestlib.FeaturePersistence,
 				},
-				{
```

Comment:
Why do we move this v1beta1 version here?

Reply:
sorry for the confusion, I changed the constant name below, maybe it's better to keep the variable name consistent. 

```diff
-	MessagingAPIVersion         = "messaging.cloud.google.com/v1alpha1"
-	MessagingV1beta1APIVersion  = "messaging.cloud.google.com/v1beta1"
+	MessagingAPIVersion         = "messaging.cloud.google.com/v1beta1"
```

Reply:
Changed back to `MessagingV1beta1APIVersion  = "messaging.cloud.google.com/v1beta1"` to make it more meaningful